src/data/registry.xml

```diff
@@ -297,6 +297,10 @@
       <data>inverse_exner_function_wrt_surface_pressure</data>
       <data>frontogenesis_function</data>
       <data>frontogenesis_angle</data>
+      <data>vertically_integrated_total_energy_of_initial_state</data>
+      <data>vertically_integrated_total_energy_of_current_state</data>
+      <data>vertically_integrated_total_water_of_initial_state</data>
+      <data>vertically_integrated_total_water_of_current_state</data>
```

Comment:
Should be `vertically_integrated_moist_air_and_condensed_water_of_current_state`

Reply:
updated!

Reply:
(updated to `vertically_integrated_water_vapor_and_condensed_water_of_current_state`)